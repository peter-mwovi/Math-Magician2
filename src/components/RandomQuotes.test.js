import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<h3>Random quotes...</h3>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
