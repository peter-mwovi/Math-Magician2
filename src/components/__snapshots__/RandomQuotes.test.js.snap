// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h3>
  Random quotes...
</h3>
`;
